### PR TITLE
[skip -ci] Feedback on PerfScore design doc

### PR DESCRIPTION
**Pull request by briansull**

### CODE REVIEW

**Comment by briansull** [comment]
Addresses the feedback from sandreenko  and CarolEidt in #26519

**Comment by briansull** [comment]
@CarolEidt @sandreenko PTAL

File: Documentation/design-docs/Perf-Score.md [hunk 1]
```diff
@@ -29,28 +29,28 @@ In order to accurately compute the actual dynamic execution cost a software mode
...
 For this implementation we won't try to model the actual def/use latencies and instead will use a simplified model to estimate the instruction latency.
 If in the future we decide to add an instruction scheduler phase to the JIT we can revisit this area.  
 I believe that it will be straightforward to add it at that time.
 
-### Simplfying Assumptions
-1. We will assume that we won't have to pay for the latency for memory store operations.  This means that we don't expect a memory read to occur that reads back the value being stored before it has finished it latency cost.
+### Simplifying Assumptions
+1. We will assume that we won't have to pay for the latency for memory store operations.  This means that we don't expect a memory read to occur that reads back the value being stored before it has finished its latency cost.
 2. We will assume that the hardware speculation will be able to hide one cycle of latency for each instruction.
 3. We won't model exact pipeline port usage and instead will model the worst case, such issuing back to back divide intructions.
```
Comment (sandreenko) [inline Documentation/design-docs/Perf-Score.md]: https://github.com/dotnet/coreclr/pull/26519#discussion_r326969183

Nit: intructions -> instructions

File: Documentation/design-docs/Perf-Score.md [hunk 2]
```diff
@@ -29,28 +29,28 @@ In order to accurately compute the actual dynamic execution cost a software mode
...
 
 ### Additional code size costs
-We also want to take into account the anount of code generated by the JIT for the method.
+We also want to take into account the amount of code generated by the JIT for the method.
 We do this as follows:
 
 1. We add 0.1 for each byte of hot code generated
```
Comment (sandreenko) [inline Documentation/design-docs/Perf-Score.md]: next line: generarted

File: Documentation/design-docs/Perf-Score.md [hunk 3]
```diff
@@ -29,28 +29,28 @@ In order to accurately compute the actual dynamic execution cost a software mode
 
 This current Perf Score feature does **NOT** propose to try to accurately model the actual dynamic execution cost.
 Instead we propose to return an estimate of the dynamic execution cost.  
-We are willing to make several simplfying assumptions that allow us to more quicly and easily return a reasonable estinmate to use as the Perf Score.
+We are willing to make several simplifying assumptions that allow us to more quicKly and easily return a reasonable estimate to use as the Perf Score.
```
Comment (sandreenko) [inline Documentation/design-docs/Perf-Score.md]: quicKly

**Comment by briansull** [comment]
@sandreenko  - While this feature can be used to examine code generated by the Perf benchmarks it isn't directly targeting them,   so I'm not sure why we would call them out specifically in this document.

**Comment by sandreenko** [comment]
> @sandreenko - While this feature can be used to examine code generated by the Perf benchmarks it isn't directly targeting them, so I'm not sure why we would call them out specifically in this document.

As I understand now if somebody does a CQ improvement it should check (after correctness of course):
1. that PerfScore is better with their changes;
2. that performance benchmarks show improvements;

and if the results of these 2 do not agree then the CQ change needs additional checks and probably fixes for PerfScore.

**Comment by briansull** [comment]
I can add a note on that.

**Pull request merged**